src/zcash/IncrementalMerkleTree.hpp

```diff
@@ -256,4 +259,86 @@ typedef libzcash::IncrementalMerkleTree<INCREMENTAL_MERKLE_TREE_DEPTH_TESTING, l
 typedef libzcash::IncrementalWitness<SAPLING_INCREMENTAL_MERKLE_TREE_DEPTH, libzcash::PedersenHash> SaplingWitness;
 typedef libzcash::IncrementalWitness<INCREMENTAL_MERKLE_TREE_DEPTH_TESTING, libzcash::PedersenHash> SaplingTestingWitness;
 
+class OrchardMerkleTree
+{
+private:
+    /// An optional incremental Sinsemilla tree (with `nullptr` corresponding to `None`).
```

Comment:
Under what conditions will this be `None`? That is, what does the `None` case represent (as opposed to the "empty tree" case)?

Reply:
Hm, in this case I guess it should actually be an error, since the frontier type is possibly-empty internally. I think that this was leftover from when I first was planning to build this around the non-empty frontier representation.